### ISSUE TITLE
Cronicle Daemon crashes if a Plugin "Executable path" does not exist.

### ISSUE THREAD

**Issue by jjtk88** [jhuckaby/Cronicle#305]
## Summary

My Cronicle daemon keeps crashing on me, I'm 99% sure it's due to my plugin path not being valid.

I have a python script that I want to run, with a relative path `./cmds/mycommand`, I created a new Plugin, put in my working directory, and typed in the above string into the `Executable`  field.  When I click "Run", nothing happens... about 5 seconds later a "Network Error 500" pops up.

When I put a valid path in the Executable field, it all works just fine.

The same crash occurs if I put a non-existent field into the "Working Directory" path as well.

Let me know what else I can do to help debug!

## Steps to reproduce the problem

Create a new plugin, input a non-existent path into the `Executable` field, Save, then Run.

### Operating system and version?

OSX Mojave 

### Node.js version?

14.4.0

### Cronicle software version?

0.8.46

### Are you using a multi-server setup, or just a single server?
Single server setup

### Are you using the filesystem as back-end storage, or S3/Couchbase?
Filesystem

### Can you reproduce the crash consistently?
Yes, seems to happen every time I type the wrong path in for an executable

## Log Excerpts

```
Tue Jul 07 2020 09:55:02 GMT-0500 (Central Daylight Time) - lpt11 - PID 22547
Error: spawn ./cmds/mycommand  ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:268:19)
    at onErrorNT (internal/child_process.js:468:16)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
```

**Comment by jhuckaby** [comment]
Thanks for this crash report!  I'll look into this as soon as possible.

**Comment by mikeTWC1984** [comment]
I had  very similar issue - daemon crashed due to plugin file format (file has CRLF new line). It will also crash if plugin is not granted with execution (I think it;s mentioned in docs). I guess any kind of plugin bug will crash the daemon, so it would probably make sence to add some try/catch to prevent it, and just report the error in job log.

**Comment by jhuckaby** [comment]
Fixed in [v0.8.47](https://github.com/jhuckaby/Cronicle/releases/tag/v0.8.47).  Thank you!
